package.json

```diff
@@ -47,10 +47,10 @@
     "vinyl-paths": "^1.0.0"
   },
   "scripts": {
-    "build": "gulp build-js && gulp build-sass",
+    "build": "gulp build-js && npm run build-legal && gulp build-sass",
```

Comment:
If `npm build` ran `npm build-legal` which ran `npm build`, _that_ would be trouble.